js/sign/package.json

```diff
@@ -43,15 +44,15 @@
     "read": "^2.0.0"
   },
   "devDependencies": {
-    "@types/node": "^14.0.0",
+    "@types/node": "^17.0.0",
     "esbuild": "^0.14.47",
     "jasmine": "^4.2.1",
     "mock-stdin": "^1.0.0",
     "prettier": "2.8.0",
     "typescript": "^4.7.3"
   },
   "engines": {
-    "node": ">= 14.0.0",
+    "node": ">= 17.0.0",
```

Comment:
Was there a reason why the upgrade of the node version was needed? The bundler plugins are also assuming >=14 node version so increasing it should be made in parallel with the other versions to make sure nothing breaks.

Also if necessary to change, the readme and release notes should also mention this.

Reply:
[KeyObject.asymmetricKeyDetails](https://nodejs.org/api/crypto.html#keyobjectasymmetrickeydetails) requires at least `15.7.0`. I've figured that it makes little sense to bump the version to something that old (since the current version is `18.19.0`), so I've ended up picking `17.0.0` as something in the middle. If you're fine with this one, I'll go ahead and update the docs too.

Reply:
Yeah fine for me as long as you upgrade the version also in the other repo :)

Reply:
Done!